src/CartGrid.C

```diff
@@ -187,3 +207,57 @@ void CartGrid::search(double *x,int *donorid,int npts)
   }
  //printf("CartGrid::search Processor %d located %d of %d points\n",myid,dcount,npts);
 }
+
+namespace {
+
+template<typename T>
+inline void create_view(TIOGA::TiogaView<T> tv, T* sptr, int sz)
```

Comment:
@sayerhs Would `TiogaMeshInfo.h` be a better place for this function ?

Reply:
`TiogaMeshInfo.h` is meant for inclusion into application codes, so I wouldn't expose a function like this in that header. I feel it should only expose the `struct` that application codes are meant to use. However, epending on how often this motif shows up elsewhere in the code, we might want to have a `gpu_utils.h` where we add this function. Right now, I feel this is too specific to what is done in `create_mesh_info()` (should have been a lambda inside that method)... but you might be right and we might eventually pull this into a utility header.